README.md

```diff
@@ -1467,7 +1489,7 @@ namespaces:
 
           - name: result
             description: The result of the operation
-            datatype: .top_level_namespace.global_error.code
+            datatype: .stdvsc.error_t
```

Comment:
We need to determine what we set the out parameters to in case an error does occur.
How do we handle this in C vs. Python. Some languages (and protocols) support optional parameters, while others do not.

Reply:
Added error handling and ranges in 6b2f9b9..c78c31d

Reply:
Thoroughly resolved.